runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerTest.java

```diff
@@ -41,19 +41,18 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 import static org.junit.Assume.assumeFalse;
-import static org.mockito.Matchers.any;
-import static org.mockito.Matchers.anyListOf;
-import static org.mockito.Matchers.eq;
-import static org.mockito.Matchers.isA;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.CALLS_REAL_METHODS;
```

Comment:
Why did we change our mocking logic here? Seems like a bunch of changes in this test file might be unrelated to the main change, could you share why you made the changes or scope them to a future PR?

Reply:
IIUC, PowerMockito is a part of PowerMock, which is an extension of Mockito. That was used for insufficient features of Mockito such as mocking static methods. Now, [mockito-inline](https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#48) can support those features. It looks like that [PowerMock project was stopped](https://github.com/powermock/powermock/issues/1117#issuecomment-1477349245).

Reply:
Got it, thank you

Reply:
Thanks for this change. It is also helpful to enable running DataflowRunner tests internally (powermockito not available in internal code base). However Mockito.mockStstic still not available internally (tracker: 233118689). Proposed to split tests involving mockStstic outside of DataflowRunnerTest in #34054